src/redux/actions/dataActions.js

```diff
@@ -159,6 +159,24 @@ export const getUserData = (userHandle) => (dispatch) => {
     });
 };
 
+export const getBookData = (userHandle) => (dispatch) => {
+  dispatch({ type: LOADING_DATA });
+  axios
+    .get(`/user/${userHandle}`)
+    .then((res) => {
+      dispatch({
+        type: SET_SCREAMS,
```

Comment:
I don't think you want to use SET_SCREAMS here.. you need to add a new SET_BOOKS call (or are you just doing this for the purpose of mockup?)